Library/Homebrew/shims/scm/git

```diff
@@ -28,9 +34,7 @@ realpath() {
     else
       path="$dir/$dest"
     fi
```

Comment:
@UniqMartin Not sure how I missed this before but if you'd prefer I change it further can you elaborate on this?